packaging/debian-sid/changelog

```diff
@@ -1,3 +1,11 @@
+snapd (2.51.3-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - interfaces/builtin: add sd-control interface (#10522)
```

Comment:
(nitpick^2) I would just remove the "(#10522) here as we generally don't include the PR number.

Reply:
ack